website1/css/components.css

```diff
@@ -26,4 +26,26 @@
   left: 16px;
   bottom: 54px;
   z-index: 1;
+}
+
+.title__section {
+  margin-left: 75px;
+  color: #4f4a45;
+}
+
+.title__header{
+  font-size: 36px;
+  height: 46px;
+}
+
+.title__line {
+  margin-left: 0;
+  margin-top: 10px;
+}
+
+.appointment__header-text{
+  width: 628px;
+  height: 44px;
+  line-height: 159%;
+  margin-top: 28px;
 }
```

Comment:
It's components? name `appointment` - relate to only the second section.
If you want to create a `title` component, create the main element and extend its modifications

Reply:
Deleted these classes and create an extendable element